BibBuddy/app/src/main/res/values/strings.xml

```diff
@@ -21,6 +21,12 @@
     <string name="back">zurück</string>
     <string name="add">+</string>
     <string name="invalid_name">Bitte einen gültigen Namen angeben. Reine Whitespaces sind nicht erlaubt.</string>
+    <string name="cancel">abbrechen</string>
+    <string name="delete">löschen</string>
+    <string name="rename">umbenennen</string>
+    <string name="change">ändern</string>
```

Comment:
And here -> change "ändern" to "Andern".

Reply:
I will leave it as it is because "Andern" is false.